spec/requests/demo_mode/sessions_spec.rb

```diff
@@ -103,9 +103,8 @@
 
             expect(DummyUser.last.name).to eq 'Tester'
             expect(response_json['id']).to eq last_session.id
-            expect(response_json['processing']).to be true
-            expect(response_json['username']).to be_nil
-            expect(response_json['password']).to be_nil
+            expect(response_json['processing']).to be false
+            expect(response_json['username']).to eq "Tester"
```

Comment:
This is the correct behavior since we wrapped the request with `perform_enqueued_jobs` above.